### DISCUSSION TITLE
Packages using cookieninja

### DISCUSSION THREAD

**Discussion by mxochicale** [UCL-ARC/python-tooling#10] [General]
As Tue 14 Mar 11:58:43 GMT 2023, I have only found https://github.com/zenml-io/zenml-project-templates which was created on Jan 31, 2023, it has ci, Slack and three maintainers. See the [github search ](https://github.com/search?q=Cookieninja&type=repositories) for Cookieninja repositories (which seems to be quite empty).

**Comment by p-j-smith** [comment]
I think it might be tricky to know which templates have been made with cookieninja in mind as they [still use a `cookiecutter.json`](https://github.com/cookieninja-generator/cookieninja#for-template-creators) for project configuration (and not e.g `cookieninja.json`). But it's probably good to keep track of what's going on with cookieninja, especially if it starts to diverge from and become incompatible with cookiecutter

**Comment by samcunliffe** [comment]
Closing this discussion (apparently I can do that! 💪 ) -- resolved via #186.
We switch back to *cutter. The ninjas disappear.
